### ISSUE TITLE
CRITICAL: loginwindow process crash

### ISSUE THREAD

**Issue by UnderEu** [w0lfschild/DarkBoot#109]
As reported on #96 and #102 , when boot screen customization is enabled, the loginwindow process freezes with 4096 threads after a couple of hours. With that, after waking up from sleep:

- OS doesn't prompt for my username and password (NOTE: FileVault is ON)
- Function keys stop working (Brightness, Volume, Keyboard Backlight and Multimedia Controls)
- System becomes unresponsive: apps start crashing, others don't open at all
- System shutdown/reboot/logout dialogs doesn't open.

Thus, forcing me to hard reboot and see everything happening again and again and again and again...
Had this issue months ago on 10.14.4 but, in that occasion, the culprit was gfxCardStatus. Switched to gSwitch and then it started happening again this weekend on 10.14.6. After an exaustive battery of tests, I confirm the culprit now IS DarkBoot.

loginwindow sample and spindump logs are attached

My system:
- 15" MacBook Pro (Mid-2012) non-Retina (MacBookPro9,1)
- macOS Mojave 10.14.6 (18G87)
- App version: 6.1.0 (Latest stable)

[Sample of loginwindow-DarkBoot_enabled.txt](https://github.com/w0lfschild/DarkBoot/files/3492837/Sample.of.loginwindow-DarkBoot_enabled.txt)
[Sample of loginwindow-Safe_Boot.txt](https://github.com/w0lfschild/DarkBoot/files/3492838/Sample.of.loginwindow-Safe_Boot.txt)
[Spindump-DarkBoot_enabled.txt](https://github.com/w0lfschild/DarkBoot/files/3492839/Spindump-DarkBoot_enabled.txt)
[Spindump-Safe_Boot.txt](https://github.com/w0lfschild/DarkBoot/files/3492840/Spindump-Safe_Boot.txt)

**Comment by w0lfschild** [comment]
Yeah, I would recommend not using that feature. If I recall correctly I also pulled that release entirely because lots of programs were detecting DarkBoot as a virus because I was using [osxinj](https://github.com/scen/osxinj) for loading into the loginwindow and that is marked as a virus by some antivirus software.

I will be looking into this at some point though and probably putting out an update around the 10.15 release after I finish up working on cDock, MacForge, and mySIMBL updates.

**Comment by w0lfschild** [comment]
If you need to fix the problem delete:

`/Library/Caches/DarkBoot`
`/Library/Application Support/SIMBL/plugins/DBLoginWindow.bundle`

Then use a version 5.X

**Comment by UnderEu** [comment]
Where do I get 5.x binary? The **_Releases_** tab only shows 4.0.5 as the latest available.

**Comment by UnderEu** [comment]
> If you need to fix the problem delete:
> 
> `/Library/Caches/DarkBoot`
> `/Library/Application Support/SIMBL/plugins/DBLoginWindow.bundle`
> 
> Then use a version 5.X

I managed to remove the app and its leftovers using AppCleaner and deleting everything related to it, system-wide (including the referenced paths). For now, no more loginwindow freezes but that white boot screen..... ARRRRRGH 😫

And where do I get 5.x binary? The Releases tab only shows 4.0.5 as the latest available.

**Comment by w0lfschild** [comment]
I don't use the releases tab really for any of my projects. All my apps archives can be found here:

https://github.com/w0lfschild/app_updates
